Assets/Scripts/UI/PauseGame.cs

```diff
@@ -0,0 +1,59 @@
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.SceneManagement;
+
+public class PauseGame : MonoBehaviour
+{
+    [SerializeField] private GameObject pauseGameUI;
+
+    private bool isOpen = false;
```

Comment:
```suggestion
    private bool isOpen;
```

false is the default state